Classes/Utility/HapticFeedback.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  HapticFeedback.swift
+//  Freetime
+//
+//  Created by Bas Broek on 24/10/2017.
+//  Copyright © 2017 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+
+enum Haptic {
+    
+    static private let selectionGenerator = UISelectionFeedbackGenerator()
+    static private let notificationGenerator = UINotificationFeedbackGenerator()
+    
+    static func triggerImpact(_ style: UIImpactFeedbackStyle) {
+        UIImpactFeedbackGenerator(style: style).impactOccurred()
+    }
+    
+    static func triggerSelection() {
+        selectionGenerator.selectionChanged()
+    }
+    
+    static func triggerNotification(_ type: UINotificationFeedbackType) {
+        notificationGenerator.notificationOccurred(type)
+    }
+}
```

Comment:
I've never understood why you have to make objects to do this stuff. I guess its for the `prepare()` method or something, but idk I don't get it.

Reply:
Yeah, I think so. Quirky APIs if you ask me